tests/Sabre/Xml/ParserTest.php

```diff
@@ -0,0 +1,587 @@
+<?php
+
+namespace Sabre\Xml;
+
+/**
+ * Test the XML reader.
+ * -Test with valid document
+ * -Test XML schema validation
+ * -Test invalid documents (missing start and end tag, malformed start and end tag and malformed quotes)
+ *
+ *
+ * PHP version 5
+ *
+ * Class ParserTest
+ * @author     Daan Biesterbos <daanbiesterbos@gmail.com>
+ * @license http://sabre.io/license/ Modified BSD License
+ * @since     1.0.0
+ */
+class ParserTest extends \PHPUnit_Framework_TestCase
```

Comment:
That sounds like a good name to me too =) Just wanted it to be a bit more descriptive.